client/src/pages/ScanStore/ScanStore.tsx

```diff
@@ -130,48 +139,91 @@ function ScanStore() {
-      <Fab
-        style={{ position: "fixed", bottom: "10px", right: "10px" }}
-        color="primary"
-        onClick={makePayment}
+    <div className="ScanStore" style={{ height: "100%" }}>
+      <Grid
+        container
+        direction="column"
+        style={{ height: "100%" }}
+        alignItems="stretch"
       >
-        <PaymentIcon />
-      </Fab>
+        <Grid item>
+          <CartHeader
+            store={curStore}
+            scanBarcodeCallback={addItemToCart}
+            content={
+              <FormGroup row>
+                <FormControlLabel
+                  control={<Switch onChange={toggleCart} color="primary" />}
+                  label="Compact View"
+                />
+                <MuiThemeProvider theme={ErrorTheme}>
+                  <FormControlLabel
+                    control={
+                      <Switch onChange={toggleDebug} color="secondary" />
+                    }
+                    label="Debug"
+                  />
+                </MuiThemeProvider>
+              </FormGroup>
+            }
+          />
+          {showDebug && [
+            <TextInputField
+              text={debugBarcode ? debugBarcode : BARCODE_PLACEHOLDER}
+              setState={setDebugBarcode}
+            />,
+            <button onClick={() => addItemToCart(debugBarcode)}>
+              Test Add Item
+            </button>,
+          ]}
+          <Divider />
+        </Grid>
+        <Grid item xs style={{ overflow: "scroll" }}>
+          <Grid item container direction="column">
+            <Grid item>
+              <Cart
+                contents={cartItems}
+                collapse={showCompactCart}
+                updateItemQuantity={updateItemQuantity}
+              />
+            </Grid>
+            <Grid item>{loadingItem && <PlaceholderCart length={1} />}</Grid>
+          </Grid>
+        </Grid>
+        <Grid item>
+          <Grid item container direction="row" justify="center">
+            <div
+              onClick={() => setShowFooter(!showFooter)}
+              style={{ zIndex: 1000, marginTop: "-36px" }}
+            >
+              <Button>
+                {showFooter ? <ExpandMoreIcon /> : <ExpandLessIcon />}
+              </Button>
+            </div>
+          </Grid>
+          <Collapse in={showFooter} unmountOnExit={true}>
+            <Grid item>
+              <Divider />
+              <CartSummary
+                cartItems={cartItems}
+                setTotalCallback={(total: number) => setCartTotal(total)}
+              />
+            </Grid>
+          </Collapse>
+          <Grid item style={{ paddingBottom: theme.spacing(2) }}>
+            <Button
+              fullWidth={true}
+              variant="contained"
+              color="primary"
+              onClick={makePayment}
+            >
+              {showFooter
+                ? "Checkout"
+                : `Checkout: $${cartTotal.toFixed(PRICE_FRACTION_DIGITS)}`}
```

Comment:
Raised issue #179 . Possibly we would need to migrate many of our 'default' strings into some `strings.json` file for fast localization - language options?

Reply:
That is a good question. 
We can treat this as a good to have feature.